neurom/apps/morph_stats.py

```diff
@@ -73,34 +83,97 @@ def _stat_name(feat_name, stat_mode):
     return '%s_%s' % (stat_mode, feat_name)
 
 
+def extract_dataframe(neurons, config):
+    """Extract stats from neurons.
+
+    Arguments:
+        neurons: a neuron, population or neurite tree
+        config (dict): configuration dict. The keys are:
+            - neurite_type: a list of neurite types for which features are extracted
+              If not provided, all neurite_type will be used
+            - neurite: a dictionary {neurite_feature: mode} where:
+                - neurite_feature is a string from NEURITEFEATURES
+                - mode is an aggregation operation provided as a string such as:
+                  ['min', 'max', 'median', 'mean', 'std', 'raw', 'total']
+
+    Returns:
+        The extracted statistics
+
+    Note:
+        An example config can be found at:
+    """
+    if isinstance(neurons, FstNeuron):
+        neurons = [neurons]
+    config = config.copy()
+    if 'neuron' in config:
+        del config['neuron']
+
+    stats = {nrn.name: extract_stats(nrn, config) for nrn in neurons}
+    columns = list(next(iter(next(iter(stats.values())).values())).keys())
```

Comment:
Lets expand here for readability.

Reply:
It's a lame line to dig into the first val of a nested dict. I can expand but I doubt it will look better anyway.
Maybe I can simply add a comment...